docs/api/java/getting-started.md

```diff
@@ -0,0 +1,831 @@
+# Getting started
+
```

Comment:
Missing intro line like in the Go page "In this guide we will demonstrate how to:"